test/rules/completed-docs/defaults/test.ts.lint

```diff
@@ -12,9 +12,12 @@ export class Aaa {
     public set prop(value) { this.bbb = value; }
     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~       [Documentation must exist for public properties.]
 
-    // TODO: TypeScript API doesn't give us a symbol for this, so we must ignore it.
+    // TypeScript API doesn't give a symbol for this before version 2.8.0
     // https://github.com/Microsoft/TypeScript/issues/14257
     [Symbol.iterator]() {}
+#if typescript >= 2.8.0
```

Comment:
TIL this is possible. Is this documented anywhere? If not, should we file a followup issue for another PR?

Reply:
not sure if it has been documented anywhere, I found it while reading the tests. I think documenting it would be useful. Let's file a follow up issue